drivers/sensor/ccs811/Kconfig

```diff
@@ -0,0 +1,69 @@
+# Kconfig - CCS811 Digital Gas sensor configuration options
+
+#
+# Copyright (c) 2018 Linaro Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig CCS811
+	bool
+	prompt "CCS811 Digital Gas Sensor"
+	depends on I2C
+	default n
+	help
+	  Enable driver for CCS811 Gas sensors.
+
+config CCS811_NAME
+	string
+	prompt "Driver name"
+	default "CCS811"
+	depends on CCS811
+	help
+	  Device name with which the CCS811 sensor is identified.
+
+config CCS811_I2C_ADDR
+	hex
+	prompt "I2C address for CCS811 Sensor"
+	depends on CCS811
+	range 0x5a 0x5b
+	default "0x5b"
+	help
+	  I2C address of the CCS811 sensor.
+	  0x5a: I2C_ADDR connected to GND.
+	  0x5b: I2C_ADDR connected to VDD.
+
+config CCS811_I2C_MASTER_DEV_NAME
+	string
+	prompt "I2C master where CCS811 is connected"
+	depends on CCS811
+	default "I2C_1"
+	help
+	  Specify the device name of the I2C master device to which the
+	  CCS811 chip is connected.
+
+config CCS811_GPIO_DEV_NAME
+	string
+	prompt "GPIO device"
+	default "GPIOC"
+	depends on CCS811
```

Comment:
Shouldn't this depend on CCS811_GPIO_WAKEUP?

Reply:
I have plans to add interrupt support also. So, just made it independent of GPIO_WAKEUP so that it can be used for both.

Reply:
Anyway will make it dependent on GPIO_WAKEUP for now!

Reply:
> I have plans to add interrupt support also. So, just made it independent of GPIO_WAKEUP so that it can be used for both.

I believe, it is more flexible to use different GPIO_DEV_NAME for wakeup and interrupt.